packages/openid4vc/src/openid4vc-verifier/OpenId4VcSiopVerifierService.ts

```diff
@@ -229,6 +232,7 @@ export class OpenId4VcSiopVerifierService {
     const requestClientId = await authorizationRequest.getMergedProperty<string>('client_id')
     const requestNonce = await authorizationRequest.getMergedProperty<string>('nonce')
     const requestState = await authorizationRequest.getMergedProperty<string>('state')
+    const responseUri = await authorizationRequest.getMergedProperty<string>('response_uri')
```

Comment:
in other code you tried both `response_uri` and `redirect_uri` is it ok to jsut take the response uri here?

Reply:
mdoc requires using response_uri. So I think we should be fine